mbgl/image/image.go

```diff
@@ -211,13 +240,13 @@ func New(
 			if err != nil {
 				// Delete the tempfile
 				img.Close()
-				return nil, err
+				return err
 			}
 			crect.length, err = img.backingStore.Write(snpImage.Data)
 			if err != nil {
 				// Delete the tempfile
 				img.Close()
```

Comment:
this could return an error. Are you ignoring it for a reason?

Reply:
What should the user do if the tempfile fails to close? It's not error, the png will still get generated. We end up if some extra temp files, which the os will clean up later? This seems to fail quite often.

Reply:
should we at least log a warning? If the system keeps running out of disk space because temp files are not getting cleaned up but the software is not warning the user it makes it more difficult to hunt down the issue.

Reply:
Also, img.Close() does not return an error. But will add the warning inside img.Close()